lib/pymedphys/_experimental/streamlit/apps/transfer_check.py

```diff
@@ -16,8 +16,12 @@
 
 from pymedphys._mosaiq import connect
 from pymedphys._streamlit import categories
+from pymedphys._streamlit.utilities.mosaiq import uncached_get_mosaiq_cursor
```

Comment:
Just import `get_mosaiq_cursor`, no need to copy the caching wrapper logic in :)